File: Prest/src/main/java/be/prest/ServiceAPI/Commentary/CommentaryUpdatePayload.java
package be.prest.ServiceAPI.Commentary;

import be.prest.ServiceAPI.Restaurant.Restaurant;
import be.prest.ServiceAPI.User.User;
import com.sun.istack.NotNull;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.sql.*;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class CommentaryUpdatePayload {
    private int id_commentary;
    private double note;
    private String text;
    private Timestamp date;
    private Restaurant restaurant;
    private User user;

}

File: frontend/prest/src/app/models/commentary/CommentaryUpdatePayload.ts
import {Payload} from "../common/Payload";
import {Restaurant} from "@restaurant/Restaurant";
import {User} from "@user/User";

export interface CommentaryUpdatePayload extends Payload
{
  id_commentary: number;
  note: number;
  text: string;
  restaurant: Restaurant;
  user: User;
  date: Date;
  actif: boolean;

}
